README.md

```diff
@@ -1 +1,12 @@
-# project-ffi-unwind
\ No newline at end of file
+# project-ffi-unwind
+
+A project to extend the Rust language to support unwinding that crosses FFI boundaries.
+
+- [What is a project?](http://smallcultfollowing.com/babysteps/blog/2019/09/11/aic-shepherds-3-0/)
+- Shepherds:
+  - [acfoltzer](https://github.com/acfoltzer)
+  - [batmanaod (Kyle)](https://github.com/batmanaod)
+  - with guidance/assistance from [nikmoatsakis](https://github.com/nikmoatsakis)
```

Comment:
```suggestion
  - with guidance/assistance from [Niko](https://github.com/nikmoatsakis)
```

I think having our "handles" and our names is probably a good idea, though I think that matters most in my case, where there's no clear connection between my name and my handle.